Main.java

```diff
@@ -0,0 +1,52 @@
+public class Main {
+ static final int SIZE =10000000;
+ static final int h = SIZE / 2;
+    public float[] calculate(float[] arr){
+        for (int i = 0; i< arr.length; i++)
+            arr[i] = (float) (arr[i] * Math.sin(0.2f + i/5)* Math.cos(0.2f +  i/5) * Math.cos(0.4f + i/2));
+        return arr;}
+        public void FirstThread() {

```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.